v2/internal/test/ca.go

```diff
@@ -230,46 +225,105 @@ func NewSerial(tb testing.TB) *big.Int {
 	return new(big.Int).SetBytes(b)
 }
 
-func WithSerial(serial *big.Int) CertificateOption {
-	return certificateOption(func(c *x509.Certificate) {
-		c.SerialNumber = serial
-	})
+type SvidOption struct {
```

Comment:
Where possible (everywhere expect protos), we should capitalize the full acronym in the name:
```suggestion
type SVIDOption struct {
```